tfe_integration_test.go

```diff
@@ -471,7 +471,7 @@ func Test_unmarshalResponse(t *testing.T) {
 		}
 		err := unmarshalResponse(responseBody, &malformattedItemStruct)
 		require.Error(t, err)
-		assert.EqualError(t, err, "v.Items must be a slice")
+		assert.EqualError(t, err, "model field \"Items\" must be a slice")
```

Comment:
Heads up that this file does not longer contain this test in releases branch. We moved this test to a new file called tfe_test.go. In this new file we are testing for helper functions.

Reply:
Whoever rebases releases-1.0.x will need to worry about this